packages/shopify-codemod/transforms/global-reference-to-import.js

```diff
@@ -68,6 +69,13 @@ export default function globalReferenceToImport(
     return fileForIdentifier[identifier];
   }
 
+  function searchForNamedExports(filename) {
+    if (filename.endsWith('.coffee'))
+      return false;
+    const namedExports = j(fs.readFileSync(filename).toString())
```

Comment:
I don't see this as a problem because there is no guarantee that ay of this will work anyways. This code breaks in this case:

```js
'expose Foo.Bar'

export function baz() {}
```

```coffee
Foo.Bar.qux();
```

but we don't really care because that's not a case that actually exists. Similarly, the simple regex might fail for something like this:

```js
'expose Foo.Bar';

const myString = `
export baz;
`; // haha my evil plan has worked!
```

Anyways, just give it a shot, see if there is a meaningful perf difference, and then we can judge whether it's worth it or not.